test/parallel/test-path.js

```diff
@@ -406,6 +406,15 @@ assert.equal(path.win32.delimiter, ';');
 // posix
 assert.equal(path.posix.delimiter, ':');
 
+// ensure ext comparison is case-insensitive on windows
+if (common.isWindows) {
```

Comment:
I think this conditional can be dropped as well, since `path.win32` would be available on all platforms and is the same implementations that would be used on real Windows.